### DISCUSSION TITLE
Utilizing alternative tunneling.

### DISCUSSION THREAD

**Discussion by S-N00B-1** [thecoder-001/MineColab#42] [Q&A] [answered]
If you are not aware already, I am dumb.
I would like to use the service playit.gg instead of Ngrok
as it supports tcp, and udp forwarding, I was wondering how I would accomplish this.

Either way thanks,
SUPERN00Bthe1

**Comment by ayoreis** [comment, marked as answer]
😀 I made a setup of this for myself some time ago, will update it and make a pull request (soon)!

For now here's the outdated version: https://colab.research.google.com/drive/1Xd0PmYvM4HAGwSNVn2uV2pXbXaqMbkmE?usp=sharing (⚠️ No idea if it still works!)
